activities/QRCode.activity/css/activity.css

```diff
@@ -10,6 +10,138 @@
 	background-image: url(../icons/photo.svg);
 }
 
+#main-toolbar #help-button {
```

Comment:
Could you review the indentation of the code you've added in this file to match the current indentation (TAB)?